CHANGELOG.rst

```diff
@@ -33,6 +33,7 @@ Improvements and new features:
  - Only remove :option:`--root` path at the start of file paths. (:issue:`452`)
  - Fix coverage report for cmake ninja builds with given in-source object-directory. (:issue:`453`)
  - Add issue templates. (:issue:`461`)
+ - Add :option:`--exclude-function-coverage` to exclude the line of the function definition in the coverage report. (:issue:`430`)  
```

Comment:
Building the sphinx docs found an error for us:

```suggestion
 - Add :option:`--exclude-function-lines` to exclude the line of the function definition in the coverage report. (:issue:`430`)  
```

Reply:
Yes, currently I'm rebasing and updating the test data.